.github/workflows/r-ci.yml

```diff
@@ -34,8 +41,8 @@ jobs:
       - name: CMake
         uses: lukka/get-cmake@latest
 
-      - name: MkVars
-        run: mkdir ~/.R && echo "CXX17FLAGS=-Wno-deprecated-declarations -Wno-deprecated" > ~/.R/Makevars
+      #- name: MkVars
```

Comment:
remove, or add a comment why this should stay -- ?

Reply:
It does not hurt anyone and shows how to set things via `MkVars` if needed.  This has value to me, so I did not delete it.